electron/main.ts

```diff
@@ -213,13 +214,16 @@ ipcMain.handle('writeFile', (event, args) => {
   )
 })
 
-ipcMain.on('lock', () =>
+let powerId = null
+ipcMain.on('lock', () => {
   writeFile(`${heroicGamesConfigPath}/lock`, '', () => 'done')
-)
+  powerId = powerSaveBlocker.start('prevent-app-suspension')
```

Comment:
Nice that was so simple. I read about this before but forgot to implement. Good that it fixed it.